contracts/ERC4626/VenusERC4626Factory.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { ERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import { ERC4626Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/ERC4626Upgradeable.sol";
+import { SafeERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import { UpgradeableBeacon } from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import { BeaconProxy } from "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
+import { Ownable2StepUpgradeable } from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import { PoolRegistryInterface } from ".././Pool/PoolRegistryInterface.sol";
+import { VenusERC4626 } from "./VenusERC4626.sol";
+import { ensureNonzeroAddress } from ".././lib/validators.sol";
+
+/// @title VenusERC4626Factory
+/// @notice Factory for creating VenusERC4626 contracts
+contract VenusERC4626Factory is Ownable2StepUpgradeable, AccessControlledV8 {
+    using SafeERC20Upgradeable for ERC20Upgradeable;
```

Comment:
Do we need this? I don't see any use of `ERC20Upgradeable` in this contract

Reply:
Removed, [df78de4](https://github.com/VenusProtocol/isolated-pools/pull/497/commits/df78de4e335ede29fe3a680af369626be05aa1e7)